tests/phpunit/integration/tests/User/Preferences.php

```diff
@@ -75,6 +93,9 @@ public function tearDown() {
 	 * @covers ::can_edit_current_user
 	 */
 	public function test_add_optin_field_to_rest_api() {
+		$this->instance->register();
+		( new Stories_Users_Controller() )->register();
```

Comment:
Why not have another class property called `$controller`, stop this repeated code.